source/bluetooth/MicroBitBLEManager.cpp

```diff
@@ -414,8 +414,11 @@ void MicroBitBLEManager::init(ManagedString deviceName, ManagedString serialNumb
 
     ble->accumulateAdvertisingPayload(GapAdvertisingData::COMPLETE_LOCAL_NAME, (uint8_t *)BLEName.toCharArray(), BLEName.length());
     ble->setAdvertisingType(GapAdvertisingParams::ADV_CONNECTABLE_UNDIRECTED);
-    ble->setAdvertisingInterval(200);
-
+    uint16_t adv_interval = 50;
```

Comment:
I think now that you've put the config in MicroBitConfig.h so we're always guaranteed to have the symbol defined and it will default to 50, this can just become
```C++
ble->setAdvertisingInterval(MICROBIT_BLE_ADVERTISING_INTERVAL);
```